pkg/store/employee/employee.go

```diff
@@ -33,6 +54,82 @@ func (s *store) All() ([]*model.Employee, error) {
 	return employees, s.db.Find(&employees).Error
 }
 
+func (s *store) Save(employee *model.Employee) error {
+	return s.db.Save(&employee).Error
+}
+
+func (s *store) Update(body EditEmployeeRequest, id string) (*model.Employee, error) {
```

Comment:
input should be `model.Employee`

Reply:
I have some in for like role which is not in the employee model

Reply:
ok